pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -47,7 +47,16 @@ export const CryptoPolicyRow = () => {
         cockpit.file("/proc/sys/crypto/fips_enabled").read()
                 .then(content => setFipsEnabled(content ? content.trim() === "1" : false));
         cockpit.file("/etc/crypto-policies/state/current")
```

Comment:
Instead of `/etc/crypto-policies/state/current`, you may want to read `/etc/crypto-policies/config`, which will be bind-mounted over by the auto-fips policy changes we've recently made.

Reply:
Note that with this PR we don't actually *read* that file any more. It's just a trigger to call `update-crypto-policies --show`. At least on older OSes, /config was the "desired" state where /state/current was the actual current state. I'm not saying I fully understand it though, I didn't write this. I'll give it a try, though!

Reply:
OK, that works and conceptually makes sense, thanks! I'll push it when we concluded the discussion in the other thread below.